pkgs/wire/dkg.go

```diff
@@ -71,6 +110,36 @@ func NewDKGProtocol(config *Config) (*dkg.Protocol, error) {
 
 	return ret, nil
 }
+func NewReshareProtocolNewNodes(config *Config) (*dkg.Protocol, error) {
+	dkgLogger := New(config.Logger)
+	dkgConfig := &dkg.Config{
+		Longterm:     config.Secret,
```

Comment:
how does the new nodes have secret?

Reply:
This id not a secret share. This a secret scalar used in DKG for encrypting/decrypting deals